Validator/LocalizedPostalCodeValidatorClosureProvider.php

```diff
@@ -96,6 +96,13 @@ public function fetchValidatorForCountry($country, $message = null)
             case 'US':
                 return $this->createValidatorClosureForConstraint($this->getMultipleRegexConstraint(['/^[0-9]{5}$/', '/^[0-9]{5}\-[0-9]{4}$/'], $message));
 
+            case 'IE':
+                $constraint = new IrelandPostcodeConstraint();
+                if ($message) {
+                    $constraint->message = $message;
+                }
+
+                return $this->createValidatorClosureForConstraint($constraint);
```

Comment:
Couldn't we do as above? rather than do this slightly differently? think it makes sense to keep the code all the same at least.